src/components/App.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import Form from './Form';
+import domToImage from 'dom-to-image';
+import fileSaver from 'file-saver';
+import Display from './Display';
+
+export default class App extends Component {
+  constructor(props) {
+    super(props);
+    this.memeRef = React.createRef(); 
+  }
+  state = {
+    header: '',
+    footer: '',
+    imgUrl: '',
+    headerColor: 'cambria',
```

Comment:
```suggestion
    headerColor: '#000000',
```
The value this was initialized to will error in the browser, make sure it is set to a valid rbg format like above to match the input type.